lib/logo/large/nrk-logo-nrk-super-on-dark.svg

```diff
@@ -0,0 +1 @@
+<svg xmlns="http://www.w3.org/2000/svg" width="55" height="24" fill="none"><path fill="#fff" d="M0 5.997h4.235v12H0v-12Zm4.49 0 2.722 12h4.61L9.541 7.923a2.296 2.296 0 0 0-.808-1.378 2.325 2.325 0 0 0-1.54-.549H4.49Zm7.587 12h4.236v-12h-4.236v12Zm9.542 0h4.236v-12h-4.236v12Zm12.667-12h-4.661c-.272.44-.516.837-.731 1.192l-.706 1.158-.74 1.208c-.238.388-.445.73-.62 1.023-.177.293-.299.49-.367.591-.17.244-.267.531-.28.828.025.3.121.59.28.845.068.101.19.298.366.592a243.264 243.264 0 0 0 1.36 2.222c.256.411.491.794.707 1.149.215.355.459.752.731 1.192h4.66l-.85-1.395c-.25-.41-.516-.845-.8-1.301a244.664 244.664 0 0 1-1.428-2.332 8.412 8.412 0 0 0-.255-.414 1.241 1.241 0 0 1-.196-.558.995.995 0 0 1 .179-.54l3.35-5.46ZM18.983 10.527a2.378 2.378 0 0 1-2.395-2.395 2.297 2.297 0 0 1 .32-1.198 2.406 2.406 0 0 1 3.272-.86c.355.208.652.504.86.86.215.362.326.776.321 1.197a2.396 2.396 0 0 1-1.18 2.075 2.296 2.296 0 0 1-1.198.32Z"/><path fill="#00B3C7" d="M49.295 3.833a8.914 8.914 0 0 1 4.731 4.73 8.67 8.67 0 0 1 .688 3.447 8.63 8.63 0 0 1-.688 3.43 8.918 8.918 0 0 1-4.73 4.731 8.676 8.676 0 0 1-3.447.688 8.67 8.67 0 0 1-3.447-.688 8.82 8.82 0 0 1-2.817-1.906 8.98 8.98 0 0 1-1.897-2.825A8.624 8.624 0 0 1 37 12.01a8.664 8.664 0 0 1 .688-3.447 8.976 8.976 0 0 1 1.897-2.825 8.816 8.816 0 0 1 2.817-1.905 8.668 8.668 0 0 1 3.447-.688 8.672 8.672 0 0 1 3.447.688Zm-.662 14.722a7.142 7.142 0 0 0 2.27-1.516 7.023 7.023 0 0 0 2.071-5.03 7.025 7.025 0 0 0-.547-2.766 7.077 7.077 0 0 0-1.524-2.27 7.197 7.197 0 0 0-2.27-1.533 6.985 6.985 0 0 0-2.784-.555 6.941 6.941 0 0 0-2.767.555 7.195 7.195 0 0 0-3.804 3.803 6.938 6.938 0 0 0-.554 2.767c-.007.95.182 1.893.554 2.767.36.846.88 1.615 1.533 2.262a7.14 7.14 0 0 0 2.27 1.516 7.025 7.025 0 0 0 2.768.547 7.07 7.07 0 0 0 2.784-.547Zm-6.654-5.55a1.303 1.303 0 0 1-.389-.945 1.344 1.344 0 0 1 .39-.978 1.282 1.282 0 0 1 .952-.398 1.356 1.356 0 0 1 .986.398 1.315 1.315 0 0 1 .406.978 1.28 1.28 0 0 1-.406.944 1.356 1.356 0 0 1-.986.398 1.283 1.283 0 0 1-.953-.398Zm6.736-2.9c.011-.034.034-.034.067 0l.398 1.077a.13.13 0 0 0 .132.1l1.176.032c.022 0 .034.009.034.025a.09.09 0 0 1-.034.058l-.845.73a.125.125 0 0 0-.083.165l.299 1.077c0 .055-.017.083-.05.083l-.978-.646a.298.298 0 0 0-.198 0l-.928.646c-.045 0-.061-.028-.05-.083l.298-1.077a.178.178 0 0 0-.083-.166l-.877-.729a.13.13 0 0 1-.017-.058c0-.016.005-.025.017-.025l1.176-.033a.141.141 0 0 0 .149-.1l.397-1.076Z"/></svg>
```

Comment:
* filen burde ha `large` i namespacet sitt, ref f.eks. `nrk-logo-large-nrk.svg`
Eksplisitt width/height burde byttes ut med viewbox, https://jakearchibald.github.io/svgomg/ kan løse deler av dette for deg